php-fpm/Dockerfile

```diff
@@ -28,23 +27,42 @@ RUN apk add --update \
     php7-tokenizer \
     php7-xml \
     php7-zlib \
-    composer \
     libsodium \
     make \
-    curl
+    curl \
+    shadow \
+    tzdata \
+    sudo \
+    zsh \
+	git \
+	bash
 
+RUN cp /usr/share/zoneinfo/Europe/Paris /etc/localtime && echo "Europe/Paris" >  /etc/timezone && apk del tzdata
 RUN rm -rf /var/cache/apk/* && rm -rf /tmp/*
 
-ADD symfony.ini /etc/php7/php-fpm.d/
-ADD symfony.ini /etc/php7/cli/conf.d/
+RUN addgroup -g 1000 www-data && adduser -u 1000 -G www-data -D www-data -s zsh
+RUN groupmod -g 1000 www-data 
+RUN echo "%sudo	ALL=(ALL:ALL) ALL" >> /etc/sudoers
+RUN usermod -u 1000 -g www-data www-data && echo "www-data:www-data" | chpasswd && addgroup sudo && adduser www-data sudo 
+
+RUN mkdir /run/php && chown -R www-data:www-data /run/php
 
-RUN addgroup www-data
-RUN adduser -u 1000 -D -g '' -G www-data www-data
+RUN chsh -s /bin/zsh www-data
+RUN sudo -u www-data sh -c "$(curl -fsSL https://raw.github.com/robbyrussell/oh-my-zsh/master/tools/install.sh)"
+RUN curl -sS https://getcomposer.org/installer | php7
+RUN mv composer.phar /usr/bin/composer
+RUN chmod +x /usr/bin/composer && composer global require hirak/prestissimo
+ENV OH_MY_ZSH_THEME="robbyrussell"
+ADD files/zshrc /home/www-data/.zshrc
 
-RUN rm /etc/php7/php-fpm.d/www.conf
-ADD symfony.pool.conf /etc/php7/php-fpm.d/
+ADD files/symfony.ini /etc/php7/php-fpm.d/
+ADD files/symfony.ini /etc/php7/cli/conf.d/
+ADD files/symfony.pool.conf /etc/php7/php-fpm.d/
 
-CMD ["php-fpm7", "-F"]
+RUN rm -f /etc/php7/php-fpm.d/www.conf
+RUN chown -R www-data:www-data /home/www-data
 
-WORKDIR /var/www/symfony
-EXPOSE 9001
+ADD files/php-fpm.sh /usr/bin/
+ENTRYPOINT ["/usr/bin/php-fpm.sh"]
```

Comment:
Leave `php-fpm` manager run as root. There is no needs to run it in user space and it's avoid some odd bugs. Our application doesn't run with privileges due to the configuration in `symfony.pool.conf`.

Reply:
in fact, it's for mac user. but ok I'll fix it